docs/detections/alert-suppression.asciidoc

```diff
@@ -71,6 +71,8 @@ image::images/alert-suppression-options.png[Alert suppression options,400]
 
 The {security-app} displays several indicators of whether a detection alert was created with alert suppression enabled, and how many duplicate alerts were suppressed.
 
+IMPORTANT: Alert suppression is only applied to alerts with an `Open` or `Acknowledged` status. To prevent interruptions or unexpected changes in suppression, avoid closing alerts that were generated during an active suppression window. This will turn off suppression for the rule and prevent future alerts from being suppressed. 
```

Comment:
```suggestion
IMPORTANT: After an alert is moved to the `Closed` status, it will no longer suppress new alerts. To prevent interruptions or unexpected changes in suppression, avoid closing alerts before the suppression interval ends.
```